README.md

```diff
@@ -1,5 +1,9 @@
 #jQuery.isOnScreen
 
+Original plugin: https://github.com/moagrius/isOnScreen
+
+I just cloned it and wrapped for UMD
```

Comment:
please remove these additions.  you can add something at the end in 3rd person like "updated for UMD by Miguel Palau <mpalauzarza@gmail.com>"